packages/plugin-react-native-unhandled-rejection/test/rejection-handler.test.ts

```diff
@@ -24,14 +24,17 @@ describe('plugin: react native rejection handler', () => {
     // in the interests of keeping the tests quick, TypeErrors get rejected quicker
     // see: https://github.com/then/promise/blob/d980ed01b7a383bfec416c96095e2f40fd18ab34/src/rejection-tracking.js#L48-L54
     try {
+      // @ts-ignore
       String.floop()
     } catch (e) {
       RnPromise.reject(e)
     }
     stop()
   })
 
-  it('should be disabled when autoDetectErrors=false', done => {
+  it('should be disabled when autoDetectErrors=false', (done) => {
+    expect.assertions(1)
+
     const c = new Client({ apiKey: 'api_key', autoDetectErrors: false })
     c._setDelivery(client => ({
       sendReport: (report) => {
```

Comment:
This fails. I think `sendReport` should have been renamed `setEvent` a while back. Looks like this one might have been missed.